AirBench/Controllers/BenchController.cs

```diff
@@ -67,13 +67,23 @@ public ActionResult Index ()
         {
             BenchRepository repository = new BenchRepository(context);
             ReviewRepository repo = new ReviewRepository(context);
+            LoginRepository Repository = new LoginRepository(context);
             List<Bench> myBenches = repository.GetBenchList();
             List<BenchList> myBenchList = new List<BenchList>();
             foreach(var bench in myBenches)
             {
+                User user = Repository.GetById(bench.CreatorUserId);
+                var name = user.Name.Split(' ');
                 var reviews = repo.GetReviewList(bench.Id);
                 BenchList currentBench = new BenchList();
                 currentBench.CreatorUserId = bench.CreatorUserId;
+                if (!(name.Count() > 1)) {
```

Comment:
When working with arrays, prefer to use the `Length` property over calling the `Count` method.